xtas/tasks/single.py

```diff
@@ -319,8 +319,8 @@ def semanticize(doc, lang='en'):
         raise ValueError("not a valid language: %r" % lang)
     text = fetch(doc)
     url = 'http://semanticize.uva.nl/api/%s?%s' % (lang,
-                                                   urlencode({'text': text}))
-    return json.loads(urlopen(url).read())['links']
```

Comment:
Ideally, yes, but I never bothered to figure out how that works in Celery. I'm not sure what kind of server is handling semanticize.uva.nl.